### ISSUE TITLE
Are there plans to supplement the code on the CSL dataset?

### ISSUE THREAD

**Issue by HW140701** [ycmin95/VAC_CSLR#17]
Thank you very much for your contribution to the community.
In the paper, I saw that experiments were carried out on both the PHOENIX14 dataset and the CSL dataset. I would like to ask if there are plans to supplement the data processing part and the training part of the code on the CSL dataset?

**Comment by ycmin95** [comment]
Thanks for your attention, we do not have clear plan to release relevant code on the CSL dataset, perhaps after the publication of the journal version. The total data process and training part are the same on different datasets (details can be found in the paper), and there is an evaluation trick on CSL due to its signer-independent setting: from my experience, using model.train() achieve better performance than model.eval().

**Comment by HW140701** [comment]
Thank you very much for your reply, looking forward to the new paper. 
I will verify the CSL dataset with reference to the details mentioned in the paper.

**Comment by hulianyuyy** [comment]
> 

I wonder if we should use 'model.train()' when evaluation. As far as i know, 'model.train()' enables backward-gradient and some special components (e.g. dropout), but seems to make no substantial change for VAC.

**Comment by mhn0411** [comment]
Subject: Follow-up Request for NMFs-CSL, CSL-Daily, and SLR500 Datasets
Dear Prof. Li, Prof. Zhou, and Research Team,
I am writing to follow up on my previous emails regarding access to your Chinese Sign Language datasets for research purposes.
My teacher had previously sent the completed and signed NMFs-CSL Dataset Release Agreement to request access to your datasets, but I have not yet received a response. I understand that you receive many requests and may be very busy with your research activities.
I am particularly interested in accessing the following datasets for my research:

NMFs-CSL Dataset (Non-Manual-Feature-Aware Isolated Chinese Sign Language Recognition Dataset)
CSL-Daily Dataset
SLR500 Dataset

My research focuses on "sign language recognition using deep learning methods" , and these datasets would be invaluable for advancing our work in this field. I am committed to using the datasets strictly for academic research purposes and will properly cite your excellent work in any resulting publications.
Could you please provide the download links for these datasets or let me know if there are any additional steps I need to complete to gain access? If there are any updated procedures or requirements, I would be happy to comply with them.
I greatly appreciate your contributions to the sign language research community and look forward to hearing from you.
Thank you for your time and consideration.
Best regards,
